documentation/usage/errorcodes/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: documentation
+menu: errorcodes
+subtitle: Error Codes
+---
+
+# Error Codes
+
+When Flyway commands fail, they throw an exception with an error code as well as a message to help you identify the problem. Below you can find the meaning of each error code and the command that causes it.
+
+## Validate Error Codes
+- `SCHEMA_DOES_NOT_EXIST` - The schema being validated against does not exist
+- `FAILED_REPEATABLE_MIGRATION` - A failed repeatable migration was detected
+- `FAILED_VERSIONED_MIGRATION` - A failed versioned migration was detected
+- `APPLIED_REPEATABLE_MIGRATION_NOT_RESOLVED` - A repeatable migration that was applied wasn't resolved in any supplied locations
+- `APPLIED_VERSIONED_MIGRATION_NOT_RESOLVED` - A versioned migration that was applied wasn't resolved in any supplied locations
+- `RESOLVED_REPEATABLE_MIGRATION_NOT_APPLIED` - A repeatable migration that was resolved has not been applied
+- `RESOLVED_VERSIONED_MIGRATION_NOT_APPLIED` - A versioned migration that was resolved has not been applied
+- `OUTDATED_REPEATABLE_MIGRATION` - An applied repeatable migration was resolved with a newer checksum and can be reapplied
+- `TYPE_MISMATCH` - The type of the resolved migration (`BASELINE`, `SQL`, `UNDO_SQL`, ...) is different from the applied migration's
```

Comment:
No need for an apostrophe to pluralize a noun.

Reply:
It's not pluralizing, it's ownership, I'm talking about the applied migration's type